phpstan.neon.dist

```diff
@@ -0,0 +1,34 @@
+# PHPStan configuration for Gutenberg
+#
+# To overload this configuration, copy this file to phpstan.neon and adjust as needed.
+#
+# https://phpstan.org/config-reference
+
+includes:
+  # The WordPress Core configuration file includes the base configuration for the WordPress codebase.
+  - tools/phpstan/base.neon
+  # The baseline file includes preexisting errors in the codebase that should be ignored.
+  # https://phpstan.org/user-guide/baseline
+  - tools/phpstan/baseline/level-0.php
+  - tools/phpstan/baseline/level-1.php
+  - tools/phpstan/baseline/level-2.php
+  - tools/phpstan/baseline/level-3.php
+  - tools/phpstan/baseline/level-4.php
+  - tools/phpstan/baseline/level-5.php
+  - tools/phpstan/baseline/level-6.php
+  - tools/phpstan/baseline/level-7.php
+
+parameters:
+  level: 7
+
+  treatPhpDocTypesAsCertain: false
+
+  ignoreErrors:
+    # @TODO: need to discover these symbols.
+    - '#Function gutenberg_.* not found.#'
```

Comment:
E.g. `gutenberg_style_engine_get_styles()`. Only thing close I could find was `wp_style_engine_get_styles()` in `packages/style-engine`.

Is there some aliasing or build step search-replace that happens? Am I missing a path in `base.neon`, or do we need to alias/stub these functions ourselves?